project/FrgaalJavaCompiler.scala

```diff
@@ -29,6 +29,9 @@ object FrgaalJavaCompiler {
   val frgaal      = "org.frgaal" % "compiler" % "19.0.1" % "provided"
   val sourceLevel = "19"
 
+  val debugArg =
+    "-J-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=localhost:8000"
```

Comment:
I believe this option shall be unified with `WithDebugCommand` setup... I wanted to mention that before, but maybe I haven't done so...

Reply:
You have mentioned that already. And the answer is buried somewhere within dozens of comments in this PR. The brief answer is that we need the `javac` process (more specifically, in this case, `java -jar frgaal-compiler.jar` process) to **wait** for the debugger and not to start without it. That is because the `java -jar frgaal-compiler.jar` process's output is piped, so the compiler process can finish even before the user notices that they could attach a debugger.

Moreover, using `WithDebugCommand.DEBUG_ARG` requires some more refactoring - for that, we would need to remove some files in `project` directory from `org.enso.build` package.

TL;DR; it would be unnecessarily complicated.